src/plugins/RepairMan.cpp

```diff
@@ -53,3 +81,13 @@ void RepairMan::OnUnitDestroyed(const sc2::Unit* unit_, Builder* builder_) {
             return;
     }
 }
+
+bool RepairMan::AssignRepairTask(const sc2::Unit& damaged_) {
+    const sc2::Point2D repair_location(damaged_.pos);
+    Worker* worker = gHub->GetClosestFreeWorker(repair_location);
+    if (!worker)
+        return false;
+
+    worker->Repair(damaged_);
+    return true;
```

Comment:
Seems that the returned value is not used anyhow and this method should be void.